package.json

```diff
@@ -35,6 +35,7 @@
     "react-native": "0.64.1",
     "react-native-android-open-settings": "1.3.0",
     "react-native-animatable": "1.3.3",
+    "react-native-base64": "0.2.1",
```

Comment:
how is this different than base64-js that is already listed in package-lock.json used by react-native itself? perhaps we should just include base64-js as a direct dependency?

There is also `base-64` used by `analytics-react-native`

Reply:
I looked in the code if we were using anything for base64, and I did not find any, so I went with a google search and installed that library. After taking a deeper look, base64-js returns a UInt8Array which is more troublesome to work with (I need a string). base-64 seems to do exactly the same as react-native-base64, so I have gone with that one.